punchcut/src/scaler.rs

```diff
@@ -0,0 +1,172 @@
+use super::{source::*, Context, Error, Hinting, NormalizedCoord, PathSink, Result, Variation};
```

Comment:
Suggest a comment that explains, or links to an explanation, what glyph scaling is, why you do it, etc. Imagine a Noogler is to work with this, would they "get it"?

Reply:
I'll add some more context. I can probably just copy from the swash docs (https://docs.rs/swash/0.1.6/swash/scale/index.html)